src/pybind/mgr/crash/module.py

```diff
@@ -114,6 +114,16 @@ def time_from_string(self, timestr):
         except ValueError:
             return datetime.datetime.strptime(timestr, OLD_DATEFMT)
 
+    def user_friendly_timestamp(self, timestr):
+        # Converts timestamp as stored in crash metadata into a more
+        # user-friendly format. See https://tracker.ceph.com/issues/43897
+        if timestr == "unknown time":
+            return timestr
+        datetime_obj = self.time_from_string(timestr)
+        return "{}+00:00".format(
+            datetime_obj.isoformat(' ', timespec='seconds')
```

Comment:
Instead of manually formatting (adding timezone) you could just properly set the timezone in `datetime_obj`:
```
datetime_obj.replace(tzinfo=datetime.timezone.utc).isoformat(' ', timespec='seconds')
```
Or probably `time_from_string` could be fixed to set timezone to utc when creating the `datetime` object.